includes/wizards/class-listings-wizard.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * Listings (Plugin) Wizard
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+use Newspack\Wizards\Traits\Admin_Header;
+use Newspack_Listings\Core as Newspack_Listings_Core;
+use Newspack_Listings\Settings as Newspack_Listings_Settings;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Easy interface for setting up info.
+ */
+class Listings_Wizard extends Wizard {
+
+	use Admin_Header;
+
+	/**
+	 * Listings plugin's Admin screen definitions (see constructor).
+	 *
+	 * @var array
+	 */
+	private $admin_screens = [];
+
+	/**
+	 * Must be run after Listings Plugin.
+	 *
+	 * @var int.
+	 */
+	protected $menu_priority = 11;
+
+	/**
+	 * Slug for current wizard screen.
+	 *
+	 * @var string
+	 */
+	private $screen_slug;
+
+	/**
+	 * Primary slug for these wizard screens.
+	 *
+	 * @var string
+	 */
+	protected $slug = 'newspack-listings';
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		
+		if ( ! is_plugin_active( 'newspack-listings/newspack-listings.php' ) ) {
```

Comment:
```suggestion
		if ( ! defined( 'NEWSPACK_LISTINGS_FILE' ) ) {
```

The path check is a bit fragile. It's not uncommon to use a different path when using a custom version for testing or other reasons.

Reply:
Agreed!